workers/social/lib/social/models/invitation.coffee

```diff
@@ -98,6 +100,29 @@ module.exports = class JInvitation extends jraphical.Module
       role          :
         type        : String
         default     : -> 'member'
+      inviterId    :
+        type        : String
+
+  @revokeInvitation: permit 'remove invitation',
+    success: (client, invite, callback) ->
+
+      { connection: { delegate:account } } = client
+      account.fetchMyPermissions client, (err, permissionAndRoles) ->
```

Comment:
when you implement https://github.com/koding/koding/pull/8692/files#r73540359 you wont need this function